docs/reference/api-v1.1.rst

```diff
@@ -16,6 +16,8 @@ What's New
 
 **New!** All controller responses now return the ``X_DEIS_PLATFORM_VERSION`` header.
 
+**New!** Users see a 404 response when modifying applications they are unauthorized to see.
+
```

Comment:
d'oh, you meant X_DEIS_API_VERSION. In the response, the patch version gets cut off (clients should not care what happens in patch releases as they're just patches), so `1.1` is intended: https://github.com/deis/deis/blob/6f00ed544d8d008ae8f5769b2581e7e1897f9249/controller/api/middleware.py#L32

No change necessary, then :)